sys/fido2/doc.txt

```diff
@@ -0,0 +1,209 @@
+ *
+ * The USB Human Interface Device (USB HID) transport binding is fully implemented.
+ *
+ * For more information about the available transport bindings refer to the [CTAP specification](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#transport-specific-bindings).
+ *
+ * **Credentials**
+ *
+ * Both types of credentials are supported. Resident and non resident.
+ *
+ *  * Resident Credentials
+ *      * Resident credentials are credentials stored on the authenticator.
+ *      * This implementation stores resident keys in flash memory.
+ *      @warning As of now the credentials (containing a private key) are stored
+ *      unencrypted in flash memory.
+ *
+ *  * Non-resident credentials
+ *      * Non-resident credentials are credentials that are stored by the relying
+ *        party in encrypted form.
+ *      * To encrypt the credentials, this implementation uses the native [AES-CCM 128
+ *        CCM implementation](https://api.riot-os.org/ccm_8h.html).
+ *
+ * **Attestation types**
+ *
+ * Currently only self attestation is supported.
+ *
+ * For more information about available attestation types refer to the [WebAuthn specification](https://www.w3.org/TR/2019/REC-webauthn-1-20190304/#sctn-attestation-types).
+ *
+ * For more information about the two types of credential refer to the WebAuthn
+ * specification (version 20190304) section 6.6.2.
+ *
+ * ### Unimplemented features
+ *
+ * **Backward compatibility with FIDO1**
+ *
+ * For more information about the backward compatibility of FIDO2 to FIDO1
+ * refer to the [CTAP spec](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#u2f-interoperability).
+ *
+ * **Support of further attestation types**
+ *
+ * Specifically these are:
+ *  * Basic Attestation
+ *  * Attestation Certificate Authority
+ *  * Elliptic Curve based Direct Anonymous Attestation
+ *
+ * For more information about available attestation types refer to the [WebAuthn specification](https://www.w3.org/TR/2019/REC-webauthn-1-20190304/#sctn-attestation-types).
+ *
+ * **Support of further transport bindings**
+ *
+ * Specifically these are:
+ *  * Near Field Communication (NFC)
+ *  * Bluetooth Low Energy (BLE)
+ *
+ * For information about the available transport bindings refer to the
+ * [CTAP spec](https://fidoalliance.org/specs/fido-v2.0-ps-20190130/fido-client-to-authenticator-protocol-v2.0-ps-20190130.html#transport-specific-bindings).
+ *
+ * **Extensions**
+ *
+ * For information about CTAP extensions refer to the
+ *
+ * **CTAP 2.1 support**
+ *
+ * None of the additions from the [CTAP 2.1 specification](https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-20210615.html)  are implemented.
+ *
+ * ### Testing
+ *
+ * For testing the FIDO2 CTAP implementation refer to the README of the test application ($(RIOTBASE)/tests/sys_fido2_ctap).
```

Comment:
Please add a note on the solokeys test package that you include (you also referenced other packages above).